src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+/* eslint-disable */
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).), Please edit App component to be class-based component.